content/applications/finance/fiscal_localizations/peru.rst

```diff
@@ -588,3 +588,265 @@ As part of the Peruvian localization, besides creating credit notes from an exis
+
+Direct Submission to SUNAT
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The creation of the GRE (Delivery Guide) in Odoo **must be sent directly to the SUNAT** regardless 
+of the electronic document provider: IAP, Digiflow or SUNAT.
+
+New Required Information
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Version 2.0 requires additional information in the general configuration as well as vehicles, contacts, 
+and products. In the general configurations, it will be necessary to add new credentials that 
+are acquired in the SUNAT portal.
+
+Flow
+****
+
+.. image:: peru/peru_flow_remision.png
+   :align: center 
+   :alt: Flow for Guia de Remision emission
+   
+Cancellations
+^^^^^^^^^^^^^
+
+Currently, the SUNAT no longer uses the term *“Anula”*, but now uses the term *“Dar de baja”*. 
+Therefore, **both** the sender and the carrier can cancel the electronic waybill, *so long as 
+the following conditions are met*: 
+
+- If the shipment has not yet been initiated
+- If the shipment has been initiated, but the receiver must be changed before reaching 
+the final destination.
+
+Testing
+^^^^^^^
+
+The SUNAT did not release any test environment; **this means that any Delivery Guides that 
+were generated by mistake will be sent to the SUNAT**.
+
+If by mistake the Waybill was created in this environment, it will be necessary to be deleted 
+from the SUNAT portal.
+
+Odoo Configuration
+******************
+
+.. important::
+   - Electronic Sender’s Waybill is currently the only supported type of Waybill in Odoo.
+   - The delivery guide is dependant on the **Inventory App**, the `l10n_pe_edi` and 
+`l10n_pe` modules.
+   
+After following the steps to configure the :doc:`Electronic invoicing <accounting_settings>` 
+and the :doc:`Master data <configura_master_data>`, go to **Apps**, **clear** any default 
+filters set, search for the `l10n_pe_edi_stock_20` module and click the *Activate* button.
+
+Next, you will need to *obtain* the Client ID and Client Secret. To do this, follow the 
+instructions in this `link <https://cpe.sunat.gob.pe/sites/default/files/inline-files/Manual_Servicios_GRE.pdf>`_.
+
+.. important::
+   Keep in mind, you must generate a second user for electronic document creation of 
+electronic documents.
+  
+To obtain the Client ID / Secret, enter the **SUNAT Portal**, and look in the “SOL” 
+menu for the **SUNAT API Credentials Management** section:
+
+.. image:: peru/menu_api_sunat.png
+   :align: center 
+   :alt: SUNAT API credential management
+
+Make sure you select the checkbox related to the **GRE issuance of receipts** and 
+the **Desktop** option in the *scope* section and save the setting:
+
+.. image:: peru/configuration_api_sunat.png
+   :align: center 
+   :alt: API Credentials registration
+
+.. important::
+   Make sure you save and accept the changes so the credentials are updated on 
+the SUNAT portal
+
+.. image:: peru/configuration_api_sunat_confirmation.png
+   :align: center 
+   :alt: API Credentials registration confirmation
```

Comment:
This section can likely be removed since it is repeating the linked [manual de servicios web plataforma nueva GRE](https://cpe.sunat.gob.pe/sites/default/files/inline-files/Manual_Servicios_GRE.pdf) instructions, no need to document instructions in a 3rd party PDF in Odoo documentation.
```suggestion
```